platform-coop-toolkit/pcc
# Move people links below the bio as per designs

## Description

This PR creates a new section below a Person's Bio called Connect. Links (as added in https://github.com/platform-coop-toolkit/pcc-framework/pull/68) are displayed in this section using an optional label for the link text or the link itself if no label is provided. Links to Twitter profiles are truncated to `@handle` and displayed with a Twitter icon.

## Steps to test

1. Add some links to a Person.
2. Visit their conference participant page.

**Expected behavior:** Links match expected view.

## Additional information

Not applicable.

## Related issues

- https://github.com/platform-coop-toolkit/pcc-framework/pull/68
